src/architecture/services.md

```diff
@@ -445,7 +445,7 @@ and push them in the queue for broadcasting, etc.
 ### REST API Initialization
 
 ```rust
-use iron::Handler;
+use exonum::api::ServiceApiBuilder;
 
 fn public_api_handler(&self, context: &ApiContext)
```

Comment:
Please update the code sample, too. As far as I understand, `wire_api` should be mentioned here.